### PR TITLE
Export configuration files and scripts to meilisearch-cloud. Use agnostic configuration

### PR DESCRIPTION
**Pull request by eskombro**
- The shell scripts that define the first configuration of a droplet have been converted to a `cloud-init` configuration, which is abstract and agnostic to cloud providers. It resides in https://github.com/meilisearch/meilisearch-cloud and it's being modified to be agnostic, so it can be used in AWS, GCP, Scaleway... and any cloud provider using `cloud-init`.

- The server part of this repository has been removed as it hasn't been used for several months

### CODE REVIEW

File: tools/build-image.py [hunk 1]
```diff
@@ -66,9 +72,9 @@
     time.sleep(5)
 
 
-# TODO: Add a check by HTTP request to IP. If fail no build.
+# # TODO: Add a check by HTTP request to IP. If fail no build.
 
-# Power down droplet
+# # Power down droplet
```
Comment (curquiza) [inline tools/build-image.py:77]:
```suggestion
# Power down droplet
```

**Review by curquiza — DISMISSED** [review]
Small useless changes (but I'm a maniac, you know me)

**Comment by eskombro** [comment]
@curquiza Thanks for your maniac review!!!

**Review by curquiza — APPROVED** [review]
💪

**Pull request merged**